src/test/java/uk/gov/hmcts/reform/sscs/service/CoverLetterServiceTest.java

```diff
@@ -189,7 +225,21 @@ private static DwpDocument getDwpDocument(String documentFilename) {
         return document;
     }
 
-    private static SscsDocument getsScsDocument(String documentFilename) {
+    private static DwpDocument getDwpDocumentEdited(String documentFilename, String editedDocumentFileName) {
```

Comment:
could this method and the method below it be combined if you use the abstract class?